.github/workflows/configlet.yml

```diff
@@ -1,19 +1,13 @@
 name: Configlet CI
 
-on:
-  workflow_dispatch:
-  push:
-    branches:
-      - main
-      - master
-  pull_request:
+on: [push, pull_request, workflow_dispatch]
 
```

Comment:
Could this respect the track config if there already is such a file?

Reply:
Ah, you mean not overwriting it? Yeah sure